src/Backoffice/Employees/App/Transformers/EmployeesTransformer.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Lightit\Backoffice\Employees\App\Transformers;
+
+use Flugg\Responder\Transformers\Transformer;
+use Lightit\Backoffice\Employees\Domain\Models\Employee;
+
+class EmployeesTransformer extends Transformer
```

Comment:
```suggestion
class EmployeeTransformer extends Transformer
```
Class names should be singular (must update the file name as well).